### PR TITLE
{data}[intel/2017b] XML-LibXML v2.0132

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2467.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/380391552ef5bd59e0468803e31f9b1c for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/fc3b7f820e3c7796cb54ea5ff96b24cb for a full test report.

File: easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0132_fix-02parse-test.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+fix failing test because of locale-related issues (?) by making it check more loosely
+author: Kenneth Hoste (HPC-UGent)
+--- XML-LibXML-2.0132/t/02parse.t.orig	2017-11-24 13:56:48.602834426 +0100
++++ XML-LibXML-2.0132/t/02parse.t	2017-11-24 13:57:37.623801916 +0100
+@@ -239,7 +239,7 @@
+     my $err_string = "$!";
+     $! = 0;
+ 
+-    my $re = qr/\ACould not create file parser context for file "\Q$badfile2\E": \Q$err_string\E/;
++    my $re = qr/Could not create file parser context for file/;
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0132_fix-02parse-test.patch]: do you need to remove `"\Q$badfile2\E": `? It's only `$err_string` that's giving issues?

Comment (boegel) [reply easybuild/easyconfigs/x/XML-LibXML/XML-LibXML-2.0132_fix-02parse-test.patch]: good point, fixed in ff34c5e1362e1563c533fd831a467c1c799644f4

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2467.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/cb84d83e79b457e98a603b4ff32e3ad4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar02.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/324b1a025fd43e9788e2325f2c85e56c for a full test report.

**Pull request merged**